tests/unit/interface-adapters/controllers/auth/sign-up.controller.test.ts

```diff
@@ -1,27 +1,17 @@
-import "reflect-metadata";
-import { afterEach, beforeEach, expect, it } from "vitest";
+import { expect, it } from 'vitest';
```

Comment:
If you want to remove the import of vitest everytime, you can use the globals:

```typescript
import { defineConfig } from 'vitest/config'

export default defineConfig({
    test: {       
        globals: true,
    },
})
```

Reply:
oh didn't know that. thanks!